main.cpp

```diff
@@ -0,0 +1,75 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+
+
+int main(int argc, char *argv[]) {
+    // std::ofsteam configfile;
+   // configfile.open("known_recipients.txt");
+
+    //std::string argument1 = "";
+
+//std::string user_input = "";
+
+//Location
+
+//if(argument = init)
```

Comment:
the argument you are looking to test against will be in `argv[1]` so you might start by `cout`ing the values of the argv array to get a handle on that.